src/Store/CompletionStoreConnector.re

```diff
@@ -14,7 +14,11 @@ module Quickmenu = Model.Quickmenu;
 
 open Actions;
 
-type lastCompletionMeet = Actions.completionMeet;
+type lastCompletionMeet = {
```

Comment:
There is still further refactoring that could be done in this file (still some duplication left between `Model.CompletionMeet.t` and `CompletionStoreConnector.lastCompletionMeet` types)